teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<ul><li>as it's -&gt; its, from -&gt; in</li><li>could have used an enum? ;)</li><li>raise on invalid molecules?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Yeah, enum would be an option, but another programming concept where is simply need a mapping. One can do it.</p><p>Regarding the "raise on invalid molecules": In principle yes, but I don't want to raise an error here and catch it below and make the whole code more complex.</p>